src/main/resources/application.yml

```diff
@@ -99,6 +99,10 @@ kafkactl:
       - "CONFIG:/spec/compatibility"
     SchemaList:
```

Comment:
`SchemaList` is not used anymore and should be removed?

Reply:
Yes, I removed it